app/src/common/shared/com/igalia/wolvic/FragmentControllerCallbacks.java

```diff
@@ -60,15 +60,16 @@ public void onStartActivityFromFragment(@NonNull Fragment fragment, Intent inten
         super.onStartActivityFromFragment(fragment, intent, requestCode, options);
     }
 
-    @Override
-    public void onStartIntentSenderFromFragment(@NonNull Fragment fragment, IntentSender intent, int requestCode, @Nullable Intent fillInIntent, int flagsMask, int flagsValues, int extraFlags, @Nullable Bundle options) throws IntentSender.SendIntentException {
-        super.onStartIntentSenderFromFragment(fragment, intent, requestCode, fillInIntent, flagsMask, flagsValues, extraFlags, options);
-    }
-
-    @Override
-    public void onRequestPermissionsFromFragment(@NonNull Fragment fragment, @NonNull String[] permissions, int requestCode) {
-        super.onRequestPermissionsFromFragment(fragment, permissions, requestCode);
-    }
+// // Deprecated method
+//    @Override
+//    public void onStartIntentSenderFromFragment(@NonNull Fragment fragment, IntentSender intent, int requestCode, @Nullable Intent fillInIntent, int flagsMask, int flagsValues, int extraFlags, @Nullable Bundle options) throws IntentSender.SendIntentException {
+//        super.onStartIntentSenderFromFragment(fragment, intent, requestCode, fillInIntent, flagsMask, flagsValues, extraFlags, options);
+//    }
+//
+//    @Override
+//    public void onRequestPermissionsFromFragment(@NonNull Fragment fragment, @NonNull String[] permissions, int requestCode) {
+//        super.onRequestPermissionsFromFragment(fragment, permissions, requestCode);
+//    }
```

Comment:
Just remove them if unused/not needed. It does not make sense to keep the code commented (we have git for history :smile: )

Reply:
Removed!